stylix/target.nix

```diff
@@ -27,14 +27,11 @@ in {
     mkOption {
       description = "Whether to style ${humanName}.";
       type = types.bool;
-
+      default = config.stylix.autoEnable && autoEnable;
+    }
+    // optionalAttrs autoEnable {
       # We can't substitute the target name into this description because some
       # don't make sense: "if the desktop background using Feh is installed"
```

Comment:
This comment is no longer necessary, since the description it talks about has been removed.

Reply:
Good catch! Closed by https://github.com/danth/stylix/pull/399/commits/4da2d6f6f3e7644ae559f9d46af8c6903bd21e9b.